cluster/cluster.go

```diff
@@ -40,6 +45,77 @@ func Join(ctx context.Context, cfg Config) (*Cluster, error) {
 	}, nil
 }
 
+type MemberAddInfo struct {
+	InitialCluster      string
+	InitialClusterState string
+}
+
+func (c *Cluster) MemberAdd(ctx context.Context, name, peerURL string) (*MemberAddInfo, error) {
+	etcdCfg := c.etcd.Config()
+	cfg := clientv3.Config{
+		Endpoints:   []string{etcdCfg.LCUrls[0].String()},
+		DialTimeout: 5 * time.Second,
+	}
+	client, err := clientv3.New(cfg)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create etcd client from config: %w", err)
+	}
```

Comment:
👍 but can we move this to the creation of the cluster and make it a var on the struct?